src/cmd/describegpt.rs

```diff
@@ -0,0 +1,118 @@
+static USAGE: &str = r#"
+Infers extended metadata about a CSV using summary statistics.
+
+Note that this command uses a LLM for inference and is therefore prone to inaccurate
+information being produced. Ensure verification of output results before using them.
+
+For examples, see https://github.com/jqnatividad/qsv/blob/master/tests/test_describegpt.rs.
+
+Usage:
+    qsv describegpt [options] [<input>]
+    qsv describegpt --help
+
+describegpt options:
+    -A, --all              Print all extended metadata options output.
+    --description          Print a general description of the dataset.
+    --dictionary           For each field, prints an inferred type, a 
+                           human-readable label, and a description.
```

Comment:
In Datapusher+, we already create a dictionary with the field name, inferred type, human-readable label, and a description.

`describegpt` should prompt the LLM  to create an expanded data dictionary.

The expanded data dictionary should at minimum, add the available summary statistics (e.g. min/max, range, avg, mad, sparsity, etc.) and try to prompt the LLM to use the stats in the description.

Reply:
@jqnatividad Thanks just a few questions:

- Would the expanded data dictionary have summary statistics be based on each field (i.e. field1 has an avg and field2 has an avg)?
- Any additional data points aside from the available ones that'd be useful in the expanded data dictionary?
- By the last part do you mean `--description` includes the overall summary statistics from `qsv stats --everything` in a description format?

Reply:
Hi @rzmk ,

> Would the expanded data dictionary have summary statistics be based on each field  (i.e. field1 has an avg and field2 has an avg)?

Yes. The expanded data dictionary will have all the stats column for each field.

> Any additional data points aside from the available ones that'd be useful in the expanded data dictionary?

For the MVP, the summary stats should be good enough.

> By the last part do you mean --description includes the overall summary statistics from qsv stats --everything in a description format?

In the last part/paragraph above, I meant the description should take into account the summary stats for the field. When I was experimenting with OpenAI GPT 3.5, it would occasionally write a field description that leveraged the stats for that field ,e.g.:

```
field: created_date 
description: The date the 311 issue was created. The oldest issue is <THE MINIMUM DATE> and the newest issue is <THE MAX DATE>
```

Reply:
Updated `--dictionary` helper message with stats clarification.